lib/netsuite/records/time_entry.rb

```diff
@@ -0,0 +1,30 @@
+module NetSuite
+    module Records
```

Comment:
Not a big deal, but all the other indentations are 2 spaces but this one has 4. Keeping indentions consistent helps with readability.